common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1142,63 +1229,71 @@ object RssConf extends Logging {
-        if (flushThread == -1) {
-          flushThread = diskType match {
-            case HDD => HDDFlusherThread(conf)
-            case SSD => SSDFlusherThread(conf)
-          }
-        }
-        (dir, maxCapacity, flushThread, diskType)
-      }
-    }.getOrElse {
-      val prefix = RssConf.workerBaseDirPrefix(conf)
-      val number = RssConf.workerBaseDirNumber(conf)
-      (1 to number).map { i =>
-        (s"$prefix$i", defaultMaxCapacity, HDDFlusherThread(conf), HDD)
-      }
-    }
-  }
+  val WORKER_FLUSHER_HDD_THREADS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.hdd.threads")
+      .withAlternative("rss.flusher.hdd.thread.count")
+      .categories("worker")
+      .doc("Flusher's thread count used for write data to HDD disks.")
+      .version("0.2.0")
+      .intConf
+      .createWithDefault(1)
 
-  def HDDFlusherThread(conf: RssConf): Int = {
-    conf.getInt("rss.flusher.hdd.thread.count", 1)
-  }
+  val WORKER_FLUSHER_SSD_THREADS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.ssd.threads")
+      .withAlternative("rss.flusher.hdd.thread.count")
+      .categories("worker")
+      .doc("Flusher's thread count used for write data to SSD disks.")
+      .version("0.2.0")
+      .intConf
+      .createWithDefault(8)
 
-  def SSDFlusherThread(conf: RssConf): Int = {
-    conf.getInt("rss.flusher.ssd.thread.count", 8)
-  }
+  val WORKER_FLUSHER_HDFS_THREADS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.hdfs.threads")
+      .withAlternative("rss.worker.hdfs.flusher.thread.count")
+      .categories("worker")
+      .doc("Flusher's thread count used for write data to HDFS.")
+      .version("0.2.0")
+      .intConf
+      .createWithDefault(4)
 
-  def diskMinimumReserveSize(conf: RssConf): Long = {
-    Utils.byteStringAsBytes(conf.get("rss.disk.minimum.reserve.size", "5G"))
-  }
+  val WORKER_FLUSHER_SHUTDOWN_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.flusher.shutdown.timeout")
+      .withAlternative("rss.worker.diskFlusherShutdownTimeoutMs")
+      .categories("worker")
+      .doc("Timeout for a flusher to shutdown.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("3s")
+
+  val WORKER_DISK_RESERVE_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.disk.reserve.size")
+      .withAlternative("rss.disk.minimum.reserve.size")
+      .categories("worker")
+      .doc("Celeborn worker reserved space for each disk.")
+      .version("0.2.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("5G")
+
+  val WORKER_FLUSHER_AVGFLUSHTIME_SLIDINGWINDOW_SIZE: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.avgFlushTime.slidingWindow.size")
+      .withAlternative("rss.flusher.avg.time.window")
+      .categories("worker")
+      .doc("The minimum flush count to enter a sliding window" +
+        " to calculate statistics about flushed time and count.")
+      .version("0.2.0")
+      .internal
```

Comment:
I think we have reached an agreement to mark internal for `celeborn.worker.flusher.avgFlushTime.slidingWindow.minCount` instead of this one in the offline discussion.

Reply:
OK. I'll change it.